file/filename.cc

```diff
@@ -24,6 +24,25 @@ namespace rocksdb {
 static const std::string kRocksDbTFileExt = "sst";
 static const std::string kLevelDbTFileExt = "ldb";
 static const std::string kRocksDBBlobFileExt = "blob";
+static const std::string kUnencryptedTempFileNameSuffix = "dbtmp.plain";
```

Comment:
```suggestion
static const std::string kUnencryptedTempFileNameSuffix = kTempFileNameSuffix + ".plain";
```
and move this nearby `kTempFileNameSuffix`?